pyproject.toml

```diff
@@ -43,12 +43,28 @@ CONDITION = "WHERE a.geom IS NOT NULL"
 GEOM = "geom_2263"
 ccp_v = "fisa_2000"
 build_schema = "public"
+CAPTURE_DATE = "2024-01-01"
+CAPTURE_DATE_PREV = "2023-07-01"
+internal_columns = '"Job_Number","Job_Type"'
+external_columns = '"Job_Number","Job_Type"'
 
 [tool.sqlfluff.templater.dbt]
 project_dir = "products/green_fast_track"
 profiles_dir = "products/green_fast_track"
 dialect = "postgres"
 
+[tool.sqlfluff.templater.jinja.context]
+source_column="bct2020"
+output_column="bct2020"
+additional_column_mappings='[("geoid", "centract2020")]'
+group_by='["boro", "bct2020", "centract2020"]'
+join_table="dcp_ct2020"
+geom_join_column="boroct2020"
+years="['2010', '2011']"
+decade="2020"
+geom="CD"
+CAPTURE_DATE="2024-01-01"
```

Comment:
(for later) I wonder how else we can handle these to prevent conflicting values with same name. maybe config files in subdirectories?

or maybe the actual value doesn't matter when linting??

Reply:
Yeah the actual value doesn't matter... UNLESS the type is different, for jinja. Luckily we don't have that issue right now, but it's certainly a possibility down the road